tests/nosetests/pyanaconda_tests/module_iscsi_test.py

```diff
@@ -61,6 +61,9 @@ def setUp(self):
         self.callback = PropertiesChangedCallback()
         self.iscsi_interface.PropertiesChanged.connect(self.callback)
 
+    def is_supported_test(self):
+        self.assertEqual(self.iscsi_interface.IsSupported(), False)
```

Comment:
Do I read this right as "_these tests will never run in a situation where iSCSI is actually configured_" ? Given the s390 expectations, it's fair I guess...

Reply:
Fixed.